db/fixtures/miq_product_features.yml

```diff
@@ -6544,7 +6557,7 @@
   - :name: Chargeback Measures
     :description: Chargeback Rate measures Accordion
```

Comment:
Do we still have an accordion? Is this feature being used at all?

Reply:
That's API specific feature, that's why it's in the API section